.gitignore

```diff
@@ -82,6 +82,10 @@ db.sqlite3-journal
 instance/
 .webassets-cache
 
+# Node stuff:
```

Comment:
Let's move this section up just below TorchGeo, most everything else in that file is from the default Python .gitignore, so I want to keep it easy to update.